src/main/java/com/hybrid/internship/library/services/BookCopyService.java

```diff
@@ -0,0 +1,27 @@
+package com.hybrid.internship.library.services;
+
+import com.hybrid.internship.library.models.BookCopy;
+
+import java.util.List;
+
+public interface BookCopyService extends AbstractService<BookCopy, Long> {
+
+    List<BookCopy> findAll();
```

Comment:
You don't need to redefine methods that are already defined in AbstractService, it defeats the purpose of abstract service. Remove them in every class that extends the AbstractService.